src/main/kotlin/exercise3/task4/TaxiParkTask.kt

```diff
@@ -10,29 +10,65 @@ package exercise3.task4
  * Find all the drivers who performed no trips.
  */
 internal fun TaxiPark.findFakeDrivers(): Set<Driver> {
-    TODO("Implement me!!!")
+    val realDriversNames = this.trips.map { it -> it.driver}.toSet()
+    val allDriversNames = this.allDrivers
+    return allDriversNames.subtract(realDriversNames)
 }
 
 /**
  * Subtask 2:
  * Find all the clients who completed at least the given number of trips.
  */
 internal fun TaxiPark.findFaithfulPassengers(minTrips: Int): Set<Passenger> {
-    TODO("Implement me!!!")
+    //test ovde ne prolazi, nisam siguran zasto
+    return this.trips
```

Comment:
The solution seems correct; the issue was in the test I pushed afterward. 
Please sync with the latest main state.